pyodide_build/recipe/graph_builder.py

```diff
@@ -130,8 +130,12 @@ def needs_rebuild(self, build_dir: Path) -> bool:
         return res
 
     def build(self, build_args: BuildArgs, build_dir: Path) -> None:
+        run_prefix = (
+            [uv_helper.find_uv_bin(), "run"] if uv_helper.should_use_uv() else []
+        )
```

Comment:
We should be careful about this, based on https://github.com/pyodide/pyodide-build/pull/132/files#r1996940827.

Reply:
Hmm, I see. Probably, we should entirely switch from subprocess to multiprocessing to avoid this kind of problem.